tests/UI/pages/FO/FObasePage.ts

```diff
@@ -240,13 +353,15 @@ class FOBasePage extends CommonPage {
    * @param symbol {string} Symbol of the currency to choose
    * @return {Promise<void>}
    */
-  async changeCurrency(page, isoCode = 'EUR', symbol = '€') {
+  async changeCurrency(page: Page, isoCode: string = 'EUR', symbol: string = '€'): Promise<void> {
```

Comment:
nitpick and out of scope but these default parameters would deserve to be inside a constant or even inside an env variable so you could change it based on your use of the tests

Reply:
I note that for the future.